modules/oe_bootstrap_theme_paragraphs/oe_bootstrap_theme_paragraphs.module

```diff
@@ -329,6 +330,64 @@ function oe_bootstrap_theme_preprocess_paragraph__oe_timeline(array &$variables)
   $variables['hide_from'] = $paragraph->get('field_oe_timeline_expand')->value;
 }
 
+/**
+ * Implements hook_preprocess_paragraph() for paragraph--oe-content-row--variant-inpage-navigation.html.twig.
+ */
+function oe_bootstrap_theme_preprocess_paragraph__oe_content_row__variant_inpage_navigation(array &$variables): void {
+  $paragraph = $variables['paragraph'];
+
+  $variables['title'] = $paragraph->get('field_oe_title')->value ?? '';
+
+  $variables['full_layout'] = TRUE;
+  $variables['scrollspy_id'] = 'bcl-inpage-navigation';
+
+  $links = [];
+  $content = [];
+
+  foreach ($paragraph->get('field_oe_paragraphs')->referencedEntities() as $delta => $sub_paragraph) {
+    /** @var \Drupal\paragraphs\Entity\Paragraph $sub_paragraph */
+    if ($sub_paragraph->hasField('field_oe_title') && !$sub_paragraph->get('field_oe_title')->isEmpty()) {
+      $links_label = $sub_paragraph->get('field_oe_title')->first()->value;
+    }
+    elseif ($sub_paragraph->hasField('field_oe_text') && !$sub_paragraph->get('field_oe_text')->isEmpty()) {
+      $links_label = $sub_paragraph->get('field_oe_text')->first()->value;
+    }
+    elseif ($sub_paragraph->hasField('field_oe_plain_text_long') && !$sub_paragraph->get('field_oe_plain_text_long')->isEmpty()) {
+      $links_label = $sub_paragraph->get('field_oe_plain_text_long')->first()->value;
+    }
+    elseif ($sub_paragraph->bundle() == 'oe_accordion') {
+      $accordion_items = $sub_paragraph->get('field_oe_paragraphs')->referencedEntities();
+      $first_accordion = $accordion_items[0];
+      $links_label = $first_accordion->get('field_oe_text')->first()->value;
```

Comment:
isEmpty?

Reply:
This one is a required field of OE Accordion item (you can check at http://localhost:8080/build/admin/structure/paragraphs_type/oe_accordion_item/fields/paragraph.oe_accordion_item.field_oe_text_long). So I didn't use the isEmpty.